packages/trader/src/App/Components/Elements/ContractAudit/contract-audit.tsx

```diff
@@ -40,6 +54,7 @@ const ContractAudit = ({
     if (!is_multiplier && !is_accumulator && !is_turbos) {
         return (
             <div className='contract-audit__wrapper'>
+                {/* @ts-expect-error: until parent component will be typescript migrated in order to verify props */}
```

Comment:
Can we also add TODOs to look into these kind of scenarios in future. 
I don't want us to miss them :)

Reply:
CI tool will catch the error with ts-expect-error and fail the pull request. TODO comment doesn't fail the pull request.